config/routes.rb

```diff
@@ -1,11 +1,31 @@
 Rails.application.routes.draw do
   root 'posts#index'
 
-  devise_for :users
+  devise_for :users,
+             controllers: {
+                 sessions: 'users/sessions',
+                 registrations: 'users/registrations'
+             }
+  get '/member-data', to: 'members#show'
+  
+  namespace :api do
+    devise_scope :user do
+      post 'login', to: '/users/sessions#create', defaults:{ format: :json}
+      post 'logout', to: '/users/sessions#destroy', defaults:{ format: :json}
+      post 'register', to: '/users/registrations#create', defaults:{ format: :json}
+    end
+    namespace :v1 do
+      resources :users, only: [:index, :show]
+      resources :posts, only: [:index, :create, :show] do
```

Comment:
- [ ] It would be better if we are able to retrieve posts from a specific user; for example; `GET : /api/users/:user_id/posts or /api/v1/users/:user_id/posts`.This should return JSON format **HINT**: We can acheive this endpoint by nesting the users resources. Like how we implemented the likes for a specific post, etc